service/src/main/java/bio/terra/workspace/service/workspace/WorkspaceService.java

```diff
@@ -81,12 +83,22 @@ public WorkspaceService(
 
   /** Create a workspace with the specified parameters. Returns workspaceID of the new workspace. */
   @Traced
-  public UUID createWorkspace(Workspace workspace, AuthenticatedUserRequest userRequest) {
+  public UUID createWorkspace(
+      Workspace workspace,
+      @Nullable ApiTpsPolicyInputs policies,
```

Comment:
Not a big issue, I'm curious why this was added in the middle.

Reply:
at one point very early on, all of the methods in `WorkspaceService` and a few other places had the `AuthenticatedUserRequest` as the last argument and I think that pattern got stuck in my brain. it's no longer true in `WorkspaceService` and it's definitely not worth the churn to re-order all the signatures, but for some reason I keep defaulting to that ordering. I'm also fine moving this to the last argument if that's cleaner

Reply:
I don't think it needs to be reordered, it was just interesting to see. Makes sense with the history.